components/FeatureHighlight/src/private/MDCFeatureHighlightView+Private.m

```diff
@@ -182,10 +182,16 @@ - (void)updateTitleFont {
   if (!_titleFont) {
     _titleFont = [MDCFeatureHighlightView defaultTitleFont];
```

Comment:
Does `defaultTitleFont` have a curve attached? I might miss it.

Reply:
It does not and that is intentional.  It will have to be migrated after the M1 curves are removed.

Reply:
Okay. Thanks for the answer.